CMakeLists.txt

```diff
@@ -35,35 +35,26 @@ cmake_minimum_required(VERSION 3.5)
 
 set(otf2xx_VERSION 1.0)
 
-set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake;${CMAKE_MODULE_PATH}")
+list(INSERT CMAKE_MODULE_PATH 0 ${CMAKE_CURRENT_LIST_DIR}/cmake)
 find_package(OTF2 8.0.1 EXACT)
 
 if (NOT OTF2_FOUND)
     message(FATAL_ERROR "Please download and install OTF2 2.1.\n"
     "Available from: http://www.vi-hps.org/upload/packages/otf2/otf2-2.1.tar.gz")
 endif()
 
-if(NOT OTF2XX_CHRONO_DURATION_TYPE)
```

Comment:
This totally weird code bloat is there for a reason. We use otf2xx typically as a git submodule and use `add_subdirectory(path/to/sub/module)` to include it into the main CMake file. This allows the main CMake file to change the default valued for these cache entries, without the need to set the cache entry exactly like the otf2xx CMakeLists.